docs/first_time_setup.md

```diff
@@ -7,50 +7,62 @@ Before starting, it's recommended that you run all of these steps [in VS Code an
+   9. Use the `flyctl` CLI tool to create a `PROMPTU_MONGODB_URL` secret with your MongoDB connection URL from `3.8.` with the following script - you have to be in the same directory as the each service's `fly.toml` (**note:** make sure you're running this from a safe environment or from within a script, as your secret will be preserved within your shell history if run directly in your shell environment):
+
+   ```sh
+   cd apps/feeder-api
+   flyctl secrets set --detach PROMPTU_MONGODB_URL="<secure value from 3.8. for the promptu-feeder user>"
+
+   cd ../../apps/db-updater
+   flyctl secrets set --detach PROMPTU_MONGODB_URL="<secure value from 3.8. for the promptu-db-updater user>"
+   ```
+
+4. **Scale Kafka memory**
+   1. Our application will need at least 2048MB (or 2GB) of memory to run. Before you run your Github workflows to deploy your applications for the first time, you have to run the following command to scale it's memory:
+   ```sh
+   cd apps/kafka
+   flyctl scale memory -a promptu-kafka-eds 2048
+   ```
+5. **Prepare Github Workflow**
+   1. In your forked Github repo, go to _Settings > Security > Secrets > Actions_ and create the following repository secrets:
+      - (**Sensitive**) `FLY_API_TOKEN` (value secured from step `2.4.`)
+   2. Update your `fly.toml` files to include the suffix you provided in step `3.4` (if you chose `paper_mache` as your suffix, then your app name will be `promptu-paper_mache` for the `ui` component and `promptu-api-paper_mache` for the `api` component)
+6. **Raise your first PR and merge it into `main` to build your infrastructure and deploy your apps** - now that all the scaffolding has been set up, it's time to add some bricks! Merging your first PR will deploy your application to fly.io for the first time! All subsequent PRs will not only deploy the latest changes to fly.io, but it will also update your infrastructure through Terraform Cloud. But we're not done yet - we still need to secure access to our DB!
+7. **Secure your promptu-api to MongoDB Atlas**
+   1. Right now, your DB can be accessed by anyone in the world! We need to restrict this so only our fly.io can communicate with it. Find out the public IP address for the applications `feeder-api` and `db-updater` by using `flyctl ssh issue` to issue an SSH key for your fly.io aap (entering `/home/vscode/.ssh/promptu-feeder-api-fly-io` and `/home/vscode/.ssh/promptu-db-updater-fly-io` respectively as your path to store the keys if you're in a devcontainer - otherwise, you have to supply the absolute path of your `~/.ssh` directory, followed by any prefix name you want e.g. `promptu-api-fly-io`) and then run `flyctl ssh console` from within the `apps/api` directory
+   2. Once inside, run the following commands in order to install `dig` and to find out your app's public IP address - record this IP address
+   ```sh
+   apt update
+   apt install -y dnsutils
+   dig +short txt ch whoami.cloudflare @1.0.0.1
+   ```
```

Comment:
Maybe we can add a: "Note: the ip address shown on the fly io UI is different than the one obtained through the dig command, please use the public ip". (If the one on the UI is the private IP, we can say that too.)

Reply:
Noted!